esx_service/cli/vmdkops_admin.py

```diff
@@ -955,9 +958,13 @@ def tenant_access_set(args):
     if args.volume_totalsize:
         volume_totalsize_in_MB = convert.convert_to_MB(args.volume_totalsize)
 
+    if not validate_create_value(str(args.allow_create)):
```

Comment:
For AdminCLI, --allow-create without explicitly specifying the value is just a special case for simplifying user input. The CLI should always pass an explicit value to auth_api. 

VMODL API will always pass an explicit value to auth_api.